### PR TITLE
Test getGlobalTrends in trends.js, add edge case code that was missed in getGlobalTrends

### PR DESCRIPTION
**Pull request by ntarn**

### CODE REVIEW

File: search-sentiments-map/test/trends-test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  describe('getGlobalTrends', function() {
+    let mockResult;
+    let mockEmpty;
+    beforeEach(() => {
+      mockResult = [
+        {trendTopic: 'Donald Trump', count: 3},
+        {trendTopic: 'Coronavirus', count: 2},
+        {trendTopic: 'Dogs', count: 1}
+      ];
+      mockEmpty = [];
```
Comment (ChenyuHeidiZhang) [inline search-sentiments-map/test/trends-test.js:17]: Why do you put those two in beforeEach()? I think you can put them in the two "it"s separately cuz they are only used by one

File: search-sentiments-map/routes/trends.js [hunk 2]
```diff
@@ -246,8 +246,10 @@ function getGlobalTrends(trendsByCountry) {
   })
 
   let globalTrends = [];
-  // Get the top 10 trends overall.
-  for (let i = 0; i < 10; i++) {
+  let numTopTrends = trendCountsArr.length < 10 ? trendCountsArr.length : 10;
```
Comment (ChenyuHeidiZhang) [inline search-sentiments-map/routes/trends.js:249]: In reality it's unlikely that there will ever be less than 10 trends in total, but I guess it's needed for the test and there's no harm to check

File: search-sentiments-map/test/trends-test.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+const mocha = require('mocha');
```
Comment (ChenyuHeidiZhang) [inline search-sentiments-map/test/trends-test.js:1]: Add the license

File: search-sentiments-map/routes/trends.js (hunk 2 above)
Comment (hayleejane3) [inline search-sentiments-map/routes/trends.js:249]: Constant for max trends.

File: search-sentiments-map/test/trends-test.js (hunk 1 above)
Comment (hayleejane3) [reply search-sentiments-map/test/trends-test.js:17]: Its convention to declare variables outside the loop, so I'd say let it be there.

Comment (carmenBen) [reply search-sentiments-map/test/trends-test.js:17]: Yeah leave it here.

File: search-sentiments-map/routes/trends.js [hunk 4]
```diff
@@ -246,7 +246,7 @@ function getGlobalTrends(trendsByCountry) {
   })
 
   let globalTrends = [];
-  let numTopTrends = trendCountsArr.length < 10 ? trendCountsArr.length : 10;
+  const numTopTrends = trendCountsArr.length < 10 ? trendCountsArr.length : 10;
```
Comment (hayleejane3) [inline search-sentiments-map/routes/trends.js:249]: I meant a const for the number 10 to represent that its the max.
